File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count =0;
        int n = nums.size();
        int count_array[n]={0};
        for (int i = 0; i<n; i++)
        {
            if (nums[i]==val)
            count ++;
            count_array[i]=count;
        }
        for(int i=0; i<n; i++)
        {
            if(nums[i]!=val)
            {
                nums[i-count_array[i]]=nums[i];
            }
        }
        return n-count;
    }
};

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-1.py
class Solution:
    def minSubArrayLen(self, target: int, nums: List[int]) -> int:
        if sum(nums)<target:
            return 0
        l, r, csum = 0,0,0
        minlen = len(nums)
        while(r<len(nums)):
            csum+=nums[r]
            r+=1
            while(csum-nums[l] >= target):
                csum-=nums[l]
                l+=1
            if csum >= target:
                minlen = min(minlen,r-l)
        return minlen